### ISSUE TITLE
AWE Support

### ISSUE THREAD

**Issue by Time6628** [KaiKikuchi/GriefPreventionPlus-SkyBlock#2]
Do you think you could add AWE support? Currently players have to /is spawn, /spawn and /is spawn again to get their islands to generate. Would be much appreciated.

**Comment by KaiKikuchi** [comment]
Sorry I had notifications off for this repository.

AWE = AsyncWorldEdit?

**Comment by Time6628** [comment]
Yes. Also, when islands are created you have to leave the chunk and teleport back for it to spawn.

**Comment by KaiKikuchi** [comment]
Are you using the latest version of GPP-Skyblock?

**Comment by Time6628** [comment]
Not sure, I'll check when I can.

**Comment by Eufranio** [comment]
soonTM kek
